src/modules/IscsiLioServer.rb

```diff
@@ -169,6 +169,18 @@ def getServiceStatus
         @statusOnStart = true
         @serviceStatus = true
       end
+      status = File.exist?("/etc/target/lio_setup.sh")
```

Comment:
I'm afraid that this will not work in installation where we use different chroot. For that reason, we use 

    if !FileUtils.Exists(filename)
      do something
    end

See, e.g., https://github.com/yast/yast-installation/blob/38f8c5fcfd53ac68c0077f3d5eaac00b995f25a2/src/include/installation/inst_inc_second.rb#L65